configs/components/puppet.rb

```diff
@@ -181,6 +181,11 @@
     end
   end
 
+  unless platform.is_windows?
```

Comment:
`pkg.link` doesn't work with Windows, and another question is should we link the executable for Windows too? There is an `eyaml.bat` file in `C:\ProgramFiles64Folder\PuppetLabs\Puppet\sys\ruby\bin`

Reply:
We usually avoid links on Windows because only users with the `SeCreateSymbolicLinkPrivilege` can create them. I'm pretty sure the security context that installs the puppet MSI has that privilege, but not sure it's guaranteed. /cc @Iristyle